apps/service-portal/src/components/Layout/Layout.css.ts

```diff
@@ -6,6 +6,9 @@ export const layoutWrapper = style({
   minHeight: `calc(100vh - ${SERVICE_PORTAL_HEADER_HEIGHT_LG}px)`,
 })
 
+export const layoutGrid = style({
+  transition: 'all 150ms ease-in-out',
```

Comment:
Might be better to specify what you're actually transitioning ... and add `will-change` as well so that the browser can optimize before the element is actually changed.

Reply:
done